Server/BackEndMonitoringServer/include/EConvertValueFromBytes.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+enum class EConvertValueFromBytes
+{
+	INTO_BYTES = 1,
+	INTO_MEGABYTES = 1048576,
```

Comment:
What about kilobytes? Also, prefer
INTO_BYTES = 1,
INTO_KILOBYTES = INTO_BYTES * 1024,
INTO_MEGABYTES = INTO_KILOBYTES * 1024,
this looks more common

Reply:
I don't see reason to add kilobytes. If accuracy is important - should use bytes. If it's not so important, megabytes and gigabytes will be enough

Reply:
Added kilobytes, changed enum values, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4)